content/02.introduction.md

```diff
@@ -2,11 +2,11 @@
 
 Pediatric brain and spinal cord tumors are collectively the second most common malignancy in children after leukemia, and they represent the leading disease-related cause of death in children [@doi:10.1093/neuonc/noz150].
 Five-year survival rates vary widely across different histologic and molecular classifications of brain tumors.
-For example, most high-grade gliomas and embryonal tumors carry a universally fatal prognosis, while children with pilocytic astrocytoma have an estimated 10-year survival rate of 92% [@doi:10.1093/neuonc/now207].
+For example, most high-grade gliomas carry a universally fatal prognosis, while children with pilocytic astrocytoma have an estimated 10-year survival rate of 92% [@doi:10.1093/neuonc/now207].
 Moreover, estimates from 2009 suggest that children and adolescents aged 0-19 with brain tumors in the United States have lost an average of 47,631 years of potential life [@doi:10.1002/cam4.410].
 
 The low survival rates for some pediatric tumors are clearly multifactorial, explained partly by our lack of comprehensive understanding of the ever-evolving array of brain tumor molecular subtypes, difficulty drugging these tumors, and the shortage of drugs specifically labeled for pediatric malignancies.
-Historically, some of the most fatal, inoperable brain tumors, such as diffuse intrinsic pontine gliomas, were not routinely biopsied due to perceived risks of biopsy and the paucity of therapeutic options that would require tissue.
+Historically, some of the most fatal, inoperable brain tumors, such as diffuse intrinsic pontine gliomas (now referred to as diffuse midline gliomas), were not routinely biopsied due to perceived risks of biopsy and the paucity of therapeutic options that would require tissue.
```

Comment:
So, this worries me. We have DIPG and DMG as _separate cancer groups_ within HGGs. This reads to me like DIPGs are now a subset of DMGs, which suggests we might need to account for this in cancer groupings? Tagging @jaclyn-taroni in case this needs further discussion, but honestly let's hope it does not 😬

Reply:
Ahhh yes actually you are right- we can take out DMG. I forgot we have some DIPGs in the paper, I think we decided way back that the DMG was reserved for K28M

Reply:
What do you mean, "we can take out DMG"? From this sentence, right? Not remove the cancer group from the paper...

Reply:
So to synthesize my understand of this conversation:

```suggestion
Historically, some of the most fatal, inoperable brain tumors, such as diffuse intrinsic pontine gliomas, were not routinely biopsied due to perceived risks of biopsy and the paucity of therapeutic options that would require tissue.
```